linux_os/guide/services/ntp/chronyd_specify_remote_server/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+
+title: 'A remote NTP server for Chrony is configured'
```

Comment:
@vojtapolasek make the title this: `Specify a Remote Time Server`

Reply:
@redhatrises Is the removal of mention of `chrony deliberate? If so, what is the reasoning?

Reply:
@yuumasato yes. It isn't needed unless you expect that multiple different time keeping technologies and rules are going to be configured which is completely unsupported and would be system breaking.

Also this is a good opportunity to add

```
requires:
    - package_chrony_installed
```

Reply:
I don't expect multiple time keeping technologies to be configured, but there are a few rules for two different components for time keeping.
In RHEL8 there is only chrony, I think.
But on RHEL7, there can be either chrony or ntpd.

Reply:
The  dependency on chrony is solved by special CPE.

Reply:
> I don't expect multiple time keeping technologies to be configured, but there are a few rules for two different time keeping technologies that can happen to be on the same DataSream.
> In RHEL8 there is only chrony, I think.
> But on RHEL7, there can be either chrony or ntpd.
> 
> Would be good to be able to differentiate which is which on the title.

Only chrony rules should be in RHEL8 and above. All other rules should be retired, so this isn't a problem.

Reply:
I can agree with being opinionated and unselect ntpd rules whenever possible.
I'm not much fond of removing Chrony from rule title.

Reply:
I am leaving it in the title and profile  can pick the appropriate rule for them.